src/g.cpp

```diff
@@ -460,39 +478,45 @@ optional<g1> g1::multiExp(std::span<const g1> points, std::span<const array<uint
     {
         for(uint64_t i = 0; i < bucketSize; i++)
         {
+            if (i & 255 == 0) {
```

Comment:
operator `==` has higher precedence than bitwise and `&`, so this test is equivalent to i & (255 == 0) which is probably not what was intended.
So either parenthesize `(i & 255)` or use `i % 256` and let the compiler optimize :-).

Reply:
change to if (yield && ((i & 255) == 0))